docs/pages/management/guides/joining-nodes-aws-ec2.mdx

```diff
@@ -185,9 +185,9 @@ Run `tctl create token.yaml` to create the token.
 ## Step 3/4. Configure your Nodes
 
 The EC2 join method can be used for Teleport services running SSH, Proxy,
-Kubernetes, Application, or Database roles. The service should be run directly
-on an AWS EC2 instance and must have network access to the AWS EC2 IMDSv2
-(enabled by default for most EC2 instances).
+Kubernetes, Application, Database, or WindowsDesktop roles. The service should
```

Comment:
Do we use the "WindowsDesktop" format elsewhere in the docs? Should we add a space there?

Reply:
The roles are actually called Kube, App, Db, and WindowsDesktop.

It looks like in this case we're referring to the roles informally and not by their actual value, so yeah I should probably just say "Windows desktop" or convert them all to their actual names. WDYT?

Reply:
works for me!